DUI3-DX/Connectors/Revit/Speckle.Connectors.RevitShared/Plugin/RevitExternalApplication.cs

```diff
@@ -69,6 +83,35 @@ public Result OnStartup(UIControlledApplication application)
     return Result.Succeeded;
   }
 
+  /// <summary>
+  /// If we directly use ex.IsFatal() in the above method, and there is an outdated version of Core loaded,
+  /// the MissingMethodException will be throw prior to executing the OnStartup method. We would prefer if
+  /// the exception is thrown after the <see cref="DllConflictManager.DetectConflicts"/> method is executed
+  /// </summary>
+  /// <param name="ex"></param>
+  /// <returns></returns>
+  private static bool IsFatal(Exception ex) => ex.IsFatal();
+
+  private void InitializePlugin(UIControlledApplication application, DllConflictManager conflictManager)
+  {
+    // POC: not sure what this is doing...  could be messing up our Aliasing????
+    //AppDomain.CurrentDomain.AssemblyResolve += OnAssemblyResolve;
+    _container = new AutofacContainer(new StorageInfo());
+    _container.PreBuildEvent += _container_PreBuildEvent;
+
+    // init DI
+    _container
+      .LoadAutofacModules(_revitSettings.ModuleFolders)
+      .AddSingletonInstance(conflictManager) // conflict detector
```

Comment:
Are we ever going to resovle this?

Reply:
Good catch, if we're only checking for DLL conflicts before the container is built, there is no need to register this class within the container 🤔

Reply:
Oh yeah I was resolving this but then I removed that code. I think we probably do want to resolve this eventually, but I can remove the registration in the meantime.